package.json

```diff
@@ -56,6 +52,11 @@
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "10.0.2",
     "babel-loader": "8.0.6",
+    "babel-plugin-transform-async-generator-functions": "6.24.1",
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "babel-plugin-transform-react-jsx": "6.24.1",
+    "babel-plugin-transform-runtime": "6.23.0",
+    "babel-preset-env": "1.7.0",
```

Comment:
I wonder how many of these are needed also.  The are all dependencies on `@wordpress/babel-preset-default`.  I know that for frontend scripts we reference the plugins directly in the build config, but we should still be able to use them without having them listed explicitly as a dependency here right (because they are included with the @wordpress package)?  I think we only need to define them here if we want to explicitly use different versions?

Reply:
Yes, I think it might be worth doing a clean-up of this packages and see which ones are required here and which ones can be omitted. Considering they were not added in this PR, how does it sound handling that in a follow-up?

Reply:
Yeah let's do in a followup 👍

Reply:
I created #829 so we don't forget about it. :slightly_smiling_face: